pkg/threescale/proxy_conf.go

```diff
@@ -233,7 +234,7 @@ func (pc *ProxyConfigCache) refreshCacheWorker(exitC chan bool) {
 			}
 		case <-refreshAfter:
 			shouldRetry := pc.refreshCache()
-			if shouldRetry && retryCounter > 0 {
+			if shouldRetry && retryCounter > 1 {
```

Comment:
so if the retry counter is 1 this won't be triggered? the question then would be, do we need to add `+1` above?